### PR TITLE
57 create service folder

### LINKED ISSUE #57: create service folder
- [x] inside service folder create product.service. js file
- [x] create call all controller methods

### PR DESCRIPTION
**Pull request by soumendrabehera99**
- [x] services/product.services.js file added

### CODE REVIEW

File: frontend/src/services/product.services.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
+// product.services.js
+import axios from "axios";
```
Comment (smruti-banty) [inline frontend/src/services/product.services.js:1]: remove comment line

File: frontend/src/services/product.services.js
Comment (smruti-banty) [inline frontend/src/services/product.services.js:1]: remove all comment line

File: frontend/src/services/product.services.js (hunk 1 above)
Comment (soumendrabehera99) [reply frontend/src/services/product.services.js:1]: comment line removed

File: frontend/src/services/product.services.js
Comment (soumendrabehera99) [reply frontend/src/services/product.services.js:1]: comment line remove

**Review by smruti-banty — APPROVED** [review]
LGTM.

**Pull request merged**